scripts/funcotator/data_sources/gnomAD/createGnomadAlleleFreqTsv.wdl

```diff
@@ -0,0 +1,114 @@
+# Create a TSV containing genomic position, dbSNP ID, alleles, and the allele frequency from v2.1 of gnomAD (hg19/b37).
+#
+# Description of inputs:
+#
+#   Required:
+#     String gatk_docker             -  GATK Docker image in which to run
+#     File gnomAD_file               -  gnomAD VCF file to process
+#     String out_file_name           -  Output file name.
+#
+#   Optional:
+#     File gatk4_jar_override        -  Override Jar file containing GATK 4.  Use this when overriding the docker JAR or when using a backend without docker.
+#     Int  mem                       -  Amount of memory to give to the machine running each task in this workflow.
+#     Int  preemptible_attempts      -  Number of times to allow each task in this workflow to be preempted.
+#     Int  disk_space_gb             -  Amount of storage disk space (in Gb) to give to each machine running each task in this workflow.
+#     Int  cpu                       -  Number of CPU cores to give to each machine running each task in this workflow.
+#     Int  boot_disk_size_gb         -  Amount of boot disk space (in Gb) to give to each machine running each task in this workflow.
+#
+# This WDL needs to decide whether to use the ``gatk_jar`` or ``gatk_jar_override`` for the jar location.  As of cromwell-0.24,
+# this logic *must* go into each task.  Therefore, there is a lot of duplicated code.  This allows users to specify a jar file
+# independent of what is in the docker file.  See the README.md for more info.
+#
+workflow CreateGnomadAlleleFreqTsv {
+
+    File gnomAD_file = "gs://gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.vcf.bgz"
```

Comment:
This is not ideal for a local or SGE backend, since it will be downloaded.  You may want to document that this script assumes running on the cloud ...

Is this script supported?

Reply:
I can add a comment at the top. 

Fundamentally if you're going to be manipulating a piece of data you should expect to have to download it.